nextcloud-startup-script.sh

```diff
@@ -217,6 +217,8 @@ download_script MENU menu
 download_script MENU server_configuration
 download_script MENU nextcloud_configuration
 download_script MENU additional_apps
+# This is needed to update the lib once during the startup script
+download_script GITHUB_REPO lib
```

Comment:
This is one step away from my plan with running the first startup without internet. Please revert.

Reply:
The other solution was better.

Reply:
If your plan is to run the whole startup script without internet, is the current implementation in master the best.
But how did you plan to get rid of run_script in every menu? 

I would suggest cloning the whole repo furing the install_script and changing run_script and download_script to use the local script if existing.

Reply:
> run_script in every menu?

Haven't figured that out yet, but maybe skip the whole menu section if the user doesn't have internet. They can always run it later.

Reply:
Of course this is also a solution, yes 👍
It might also be much easier to implement